# Robot name goes here
car: # change the name
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  fr_steer_joint_effort_controller: # change the name
    type: effort_controllers/JointPositionController  # Type of controller. 
    joint: fr_steer_joint # Name of the joint.  
    pid : {p: 100.0 , i: 1 , d: 0.5} # Enter your pid values eg. {p: 100.0 , i: 0.0 , d: 0.0} 

  # Controller 2
  fl_steer_joint_effort_controller: # change the name
    type: effort_controllers/JointPositionController  # Type of controller. 
    joint: fl_steer_joint # Name of the joint.  
    pid : {p: 100.0 , i: 1 , d: 0.5}
  # Controller 3
  rear_axle_joint_velocity_controller: # change the name
    type: effort_controllers/JointEffortController  # Type of controller. 
    joint: rear_axle_joint # Name of the joint.  
    pid : {p: 10.0 , i: 0.5 , d: 0.2}
  # Controller 4
  fr_wheel_joint_velocity_controller: # change the name
    type: effort_controllers/JointEffortController  # Type of controller. 
    joint: fr_wheel_joint # Name of the joint.  
    pid : {p: 10.0 , i: 0.5 , d: 0.2}
  # Controller 5
  fl_wheel_joint_velocity_controller: # change the name
    type: effort_controllers/JointEffortController  # Type of controller. 
    joint: fl_wheel_joint # Name of the joint.  
    pid : {p: 10.0 , i: 0.5 , d: 0.2}

